cmd/main.go

```diff
@@ -74,6 +70,7 @@ func main() {
 	// Create DDB service client
 	ddbSvc := dynamodb.NewFromConfig(cfg)
 
+	log.Printf("Service clients created")
```

Comment:
NIT: do you think it's helpful for users to keep this log statement? I guess it was used for debugging. I am fine with keeping it, just asking :)

Reply:
Yes I left those because they were helpful in getting an idea where the issue came up, at least for me it was handy. Figured it doesn't hurt to leave them there